pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -19,33 +19,243 @@ package dockershim
 import (
 	"fmt"
 
+	dockertypes "github.com/docker/engine-api/types"
+	dockercontainer "github.com/docker/engine-api/types/container"
+	dockerfilters "github.com/docker/engine-api/types/filters"
+
 	runtimeApi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+const (
+	defaultSandboxImage = "gcr.io/google_containers/pause-amd64:3.0"
+
+	// Various default sandbox resources requests/limits.
```

Comment:
oom score: https://github.com/kubernetes/kubernetes/blob/ef0c9f0c5b8efbba948a0be2c98d9d2e32e0b68c/pkg/kubelet/qos/policy.go#L24

cpu share: https://github.com/kubernetes/kubernetes/blob/59ca5986dd8cba39c4933160c11896c401e58a4f/pkg/kubelet/dockertools/docker.go#L373

but you're right, there are no default values for cpu quota and period:
https://github.com/kubernetes/kubernetes/blob/59ca5986dd8cba39c4933160c11896c401e58a4f/pkg/kubelet/dockertools/docker.go#L350
I'll fix them.